docs: add semantic line break guidance to style conventions (#285)
grafana/writers-toolkit: docs/sources/write/style-guide/style-conventions/index.md

By disabling authentication requirements, anyone can access your Grafana instance.
There is a considerable security risk associated with this.
{{% /admonition %}}

## Semantic line breaks

> We use [these general rules](https://sembr.org/) for adding line breaks in your writing, but we recommend starting a new line after each sentence to make the text easier to review and edit.

Reviewer: Given that we don't ourselves follow these rules in our documentation, may I suggest we soften this a bit:

```suggestion
The [Semantic Line Breaks organization](https://sembr.org/) suggests adding line breaks in your writing. We find this can help with reviews.

Note that we don't require that this suggestion be followed. We share the Semantic Line Breaks rules so that you can discuss with your review team and adapt to your circumstance.
```

Reviewer 2: Fiona, I agree with your phrasing here very much. The phrase "[Grafana Writers' Toolkit] recommends" is a powerful oe, and as I think I've made clear we probably shouldn't be throwing our weight behind the SLB spec that we don't use in any of our docs or writer's toolkit. But I think it's good that we give people the option to use it whenever it can add clarity to the review process.

With line breaks:

```markdown
When you write in active voice, you identify the subject of the sentence and the action that the subject performs.
For example, “John drove the car” is active voice because it is clear that John (the subject) performed an action (drove).
The passive voice variation is “The car was driven by John.”
```

Without line breaks:

```markdown
When you write in active voice, you identify the subject of the sentence and the action that the subject performs. For example, “John drove the car” is active voice because it is clear that John (the subject) performed an action (drove). The passive voice variation is “The car was driven by John.”
```

> The output is the same in both cases. However, the first is easier to review and edit and is less subject to the screen and text editor settings of each contributor.

Author: @jdbaldry - I can't figure out what prettier wants me to do here.

> The output is the same in both cases. However, the first is easier to review and edit and is less subject to the screen and text editor settings of each contributor.

Reviewer 3: I've updated the suggestion and committed it.

By disabling authentication requirements, anyone can access your Grafana instance.
There is a considerable security risk associated with this.
{{% /admonition %}}

## Semantic line breaks

> The [Semantic Line Breaks organization](https://sembr.org/) suggests adding line breaks in your writing. We find this can help with reviews.

Author: ```suggestion
The [Semantic Line Breaks organization](https://sembr.org/) suggests adding line breaks in your writing. 
We find this can help with reviews.
```

> Note that we don't require that this suggestion be followed. We share the Semantic Line Breaks rules so that you can discuss with your review team and adapt to your circumstance.

Reviewer 3: nit: Not using semantic line breaks in the PR about semantic line breaks :P

> Note that we don't require that this suggestion be followed. We share the Semantic Line Breaks rules so that you can discuss with your review team and adapt to your circumstance.

Author: ```suggestion
Note that we don't require that this suggestion be followed. 
We share the Semantic Line Breaks rules so that you can discuss with your review team and adapt to your circumstance.
```

With line breaks:

```markdown
When you write in active voice, you identify the subject of the sentence and the action that the subject performs.
For example, “John drove the car” is active voice because it is clear that John (the subject) performed an action (drove).
The passive voice variation is “The car was driven by John.”
```

Without line breaks:

```markdown
When you write in active voice, you identify the subject of the sentence and the action that the subject performs. For example, “John drove the car” is active voice because it is clear that John (the subject) performed an action (drove). The passive voice variation is “The car was driven by John.”
```

> The output is the same in both cases. However, the first is easier to review and edit and is less subject to the screen and text editor settings of each contributor.

Author: ```suggestion
The output is the same in both cases. 
However, the first is easier to review and edit and is less subject to the screen and text editor settings of each contributor.
```
